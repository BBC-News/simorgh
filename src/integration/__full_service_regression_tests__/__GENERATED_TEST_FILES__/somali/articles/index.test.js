/**
 * @service somali
 * @pathname /somali/articles/cgn6emk3jm8o
 *
 * THIS TEST SUITE WAS GENERATED BY A BUILD SCRIPT. DO NOT EDIT THIS FILE.
 */

import runA11yTests from '../../../../common/a11y';
import runHeaderTests from '../../../../common/header';
import runFootertests from '../../../../common/footer';
import runSEOtests from '../../../../common/SEO';
import runPerformanceTests from '../../../../common/performance';
runA11yTests();
runHeaderTests();
runFootertests();
runSEOtests();
runPerformanceTests();
it('Image with caption', () => {
  const imageEl = document.querySelector(
    'main figure img, main figure amp-img',
  );
  const imageCaptionEl = document.querySelector('main figure figcaption');
  expect(imageEl).toBeInTheDocument();
  expect(imageEl).toBeTruthy();
  expect(imageEl.getAttribute('src')).toMatchInlineSnapshot(
    `"https://ichef.bbci.co.uk/news/640/cpsdevpb/9d53/test/ad9ec840-de13-11e9-b23c-f1ac9644bb55.jpg"`,
  );
  expect(imageCaptionEl).toBeInTheDocument();
  expect(imageCaptionEl.textContent).toBeTruthy();
  expect(imageCaptionEl.textContent).toMatchInlineSnapshot(
    `"Qoraalka sawirka, Sawirkan tijaabada ah, oo ah xuquuqda daabacaadda BBC, waxay muujineysaa khariidad Faransiis ah. Muuqaalku wuxuu ku jiraa saddexda qaybood ee ugu horreeya wuxuuna leeyahay cinwaankan."`,
  );
});
