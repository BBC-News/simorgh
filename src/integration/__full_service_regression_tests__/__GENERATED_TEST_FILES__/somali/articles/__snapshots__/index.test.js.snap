// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical link 1`] = `"http://localhost:7080/somali/articles/cgn6emk3jm8o"`;

exports[`Dir attribute 1`] = `"ltr"`;

exports[`FB admins 1`] = `"100004154058350"`;

exports[`FB app ID 1`] = `"1609039196070050"`;

exports[`Footer Copyright 1`] = `"Â© 2020 BBC. BBC masuul kama ahan macluumadka bogagga kale ee dibadda. Akhri xogta ku saabsan sida aan u abaarno bogagga dibadda."`;

exports[`Header Branding 1`] = `"BBC News, Somali"`;

exports[`Heading level 1 1`] = `"Tani waa cinwaanka maqaalkan imtixaanka"`;

exports[`Lang attribute 1`] = `"so"`;

exports[`OG description 1`] = `"Tani waa cinwaanka SEO ee maqaalkan tijaabada ah"`;

exports[`OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/somali.png"`;

exports[`OG image alt 1`] = `"BBC News Somali"`;

exports[`OG locale 1`] = `"so-SO"`;

exports[`OG site name 1`] = `"BBC News Somali"`;

exports[`OG title 1`] = `"Tani waa cinwaanka SEO ee maqaalkan tijaabada ah - BBC News Somali"`;

exports[`OG type 1`] = `"article"`;

exports[`OG url 1`] = `"http://localhost:7080/somali/articles/cgn6emk3jm8o"`;

exports[`Page title 1`] = `"Tani waa cinwaanka SEO ee maqaalkan tijaabada ah - BBC News Somali"`;

exports[`Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Skip to content link 1`] = `"U gudub qaybta macluumaadka"`;

exports[`Twitter card 1`] = `"summary_large_image"`;

exports[`Twitter creator 1`] = `"@bbcsomali"`;

exports[`Twitter description 1`] = `"Tani waa cinwaanka SEO ee maqaalkan tijaabada ah"`;

exports[`Twitter image alt 1`] = `"BBC News Somali"`;

exports[`Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/somali.png"`;

exports[`Twitter site 1`] = `"@bbcsomali"`;

exports[`Twitter title 1`] = `"Tani waa cinwaanka SEO ee maqaalkan tijaabada ah - BBC News Somali"`;
